src/cdk/testing/tests/BUILD.bazel

```diff
@@ -29,16 +30,29 @@ ts_library(
     ],
 )
 
+ng_test_library(
```

Comment:
I'm new to bazel, but as far as I understood, I need a subfolder (called it ```env_independent```) to create a shared artifact. Didn't like to put the spec file into the ```harnesses```-folder (as it isn't a harness) but maybe we should move the ```harnesses```-folder into ```env-independent```as they are for both environments, too

Reply:
Actually you don't need a separate directory, you just need to make sure the globs include/exclude the appropriate files. We do something kind of similar to this with our tests for the harnesses for individual components. We share one set of specs between the current and new MDC-based version of the component (see https://github.com/angular/components/blob/master/src/material/checkbox/testing/BUILD.bazel#L23 as an example)

Reply:
Tried it that way, didn't work, now I tried it again, did work - maybe I had a typo.

Renamed the file to differentiate from those shared specs so it's more obviously that it's only shared between protractor and testbed - not with MDC

Reply:
but causes a linting error without description - any hint?

Reply:
I think you can just run `yarn format` to auto-fix lint issues with the bazel files

Reply:
glad its fixed, but sounds like we could use better error messages for when these things fail